imports/protobuf/session.proto

```diff
@@ -11,6 +11,7 @@ enum SessionInitializationBehavior {
   SESSION_INITIALIZATION_BEHAVIOR_UNSPECIFIED = 0;
   SESSION_INITIALIZATION_BEHAVIOR_INITIALIZE_NEW = 1;
   SESSION_INITIALIZATION_BEHAVIOR_ATTACH_TO_EXISTING = 2;
+  SESSION_INITIALIZATION_BEHAVIOR_REFERENCE_COUNTED = 3;
```

Comment:
This conflicts with NI-VISA's extensions to this enum.

https://github.com/ni/grpc-device/wiki/Sharing-Driver-Sessions-Between-Clients